packages/daimo-api/src/contract/nameRegistry.ts

```diff
@@ -99,6 +104,10 @@ export class NameRegistry {
       hash,
     });
     if (tx.status !== "success") throw new Error("Transaction failed");
+
+    // Cache
+    this.cacheAccount({ name, addr: address });
```

Comment:
fixes a bug where newly created accounts wouldn't show up in search until API restart